pages/partners/index.js

```diff
@@ -8,23 +11,114 @@ const partners = [
         name: 'Covid India Campaign',
         logoURL: 'https://covidindiacampaign.org/wp-content/uploads/2020/05/logo-2-1.png',
         url: 'http://www.covidindiacampaign.org/'
+    },
+    {
+        id: '2',
+        name: 'PARLIAMENTARIANS WITH INNOVATORS FOR INDIA',
+        logoURL: 'https://ik.imagekit.io/img1991/PIINDIA/shared/pi_india_logo_yellow_8eQILlgiD.svg',
+        url: 'https://piindia.org/index-web.html'
+    },
+    {
+        id: '3',
+        name: 'Step one',
+        logoURL: 'https://projectstepone.org/wp-content/uploads/2020/10/StepOne-logo-300x83.png',
+        url: 'https://projectstepone.org/'
+    },
+    {
+        id: '4',
+        name: 'Swasth',
+        logoURL: 'https://www.swasth.app/static/media/logo.9c8319e6.svg',
+        url: 'https://www.swasth.app/home'
+    },
+    {
+        id: '5',
+        name: 'Coronasafe',
+        logoURL: 'https://assets.zyrosite.com//Yyve4DOKXjcZ8bJ0/coronasafe-logo-m5KOkkg54nIzwx6x.svg',
+        url: 'https://piindia.org/index-web.html'
+    },
+    {
+        id: '6',
+        name: 'India covid resources',
+        logoURL: 'https://res.cloudinary.com/glide/image/fetch/f_auto,w_150,h_150,c_lfill/https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Fglide-prod.appspot.com%2Fo%2Ficon-images%252Fanonymous-845cd98e-e571-483a-984b-daebd749be79.png%3Falt%3Dmedia%26token%3Db8da4311-19b4-49c5-868c-dc46d73b1e0f',
+        url: 'https://indiacovidresources.in/'
+    },
+    {
+        id: '7',
+        name: 'The product folks',
+        logoURL: 'https://static.wixstatic.com/media/7c7963_a199f541090c4734a25b732b94f1ad3a~mv2.png/v1/fill/w_135,h_79,al_c,q_85,usm_0.66_1.00_0.01/TPF_Logo.webp',
+        url: 'https://www.theproductfolks.com/'
     }
 ];
 
 const Partners = () => {
+    const SEO = {
```

Comment:
SEO isnt implemented here, you gotta pass it in <NextSEO />

Also Description can be the text  in the pages instead of Title

Reply:
I am pushing it